app/views/home/index.html.erb

```diff
@@ -6,12 +6,40 @@
   </p>
 </div>
 
-<h2><%= t('latest_projects') %></h2>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-4">
+      <h2><%= t('latest_projects') %></h2>
 
-<ul>
-  <% cache do %>
-    <% @latest_projects.each do |project| %>
-      <li><%= link_to(project.name, project_path(project.id)) %></li>
-    <% end %>
-  <% end %>
-</ul>
+      <ul>
+        <% cache do %>
```

Comment:
I just tested and the current code is broken. Notice how Kalibro Configuration Attributes are being looked up even when they're supposed to be cached. Notice how three cache lookups with the same keys are made, when they should be different as the contents are different.

The [page also doesn't show any configuration at all](http://i.imgur.com/Zmx9LRC.png). All three boxes are empty. 

```
$ rm -rf tmp/cache/*
$ rails c
2.3.0 :003 > KalibroConfiguration.latest(5).count
 => 4
2.3.0 :004 > Project.latest(5).count
 => 0
2.3.0 :005 > Repository.latest(5).count
 => 0 
$ rails s
Started GET "/pt" for 127.0.0.1 at 2016-04-14 12:01:48 -0300
Processing by HomeController#index as HTML
  Parameters: {"locale"=>"pt"}
  KalibroConfigurationAttributes Load (0.3ms)  SELECT  "kalibro_configuration_attributes".* FROM "kalibro_configuration_attributes" WHERE "kalibro_configuration_attributes"."kalibro_configuration_id" = $1 LIMIT 1  [["kalibro_configuration_id", 4]]
  KalibroConfigurationAttributes Load (0.2ms)  SELECT  "kalibro_configuration_attributes".* FROM "kalibro_configuration_attributes" WHERE "kalibro_configuration_attributes"."kalibro_configuration_id" = $1 LIMIT 1  [["kalibro_configuration_id", 3]]
  KalibroConfigurationAttributes Load (0.2ms)  SELECT  "kalibro_configuration_attributes".* FROM "kalibro_configuration_attributes" WHERE "kalibro_configuration_attributes"."kalibro_configuration_id" = $1 LIMIT 1  [["kalibro_configuration_id", 2]]
  KalibroConfigurationAttributes Load (0.2ms)  SELECT  "kalibro_configuration_attributes".* FROM "kalibro_configuration_attributes" WHERE "kalibro_configuration_attributes"."kalibro_configuration_id" = $1 LIMIT 1  [["kalibro_configuration_id", 1]]
  Cache digest for app/views/home/index.html.erb: bd6b86e43ee3144d2e082252dd8d607d
Read fragment views/localhost:3000/pt/bd6b86e43ee3144d2e082252dd8d607d (0.1ms)
Write fragment views/localhost:3000/pt/bd6b86e43ee3144d2e082252dd8d607d (9.1ms)
  Cache digest for app/views/home/index.html.erb: bd6b86e43ee3144d2e082252dd8d607d
Read fragment views/localhost:3000/pt/bd6b86e43ee3144d2e082252dd8d607d (0.0ms)
  Cache digest for app/views/home/index.html.erb: bd6b86e43ee3144d2e082252dd8d607d
Read fragment views/localhost:3000/pt/bd6b86e43ee3144d2e082252dd8d607d (0.0ms)
  Rendered home/index.html.erb within layouts/application (12.8ms)
  Rendered shared/_navbar.html.erb (6.3ms)
  Rendered shared/_footer.html.erb (20.3ms)
Completed 200 OK in 2641ms (Views: 2620.5ms | ActiveRecord: 0.8ms)
...
Started GET "/pt" for 127.0.0.1 at 2016-04-14 12:02:11 -0300
Processing by HomeController#index as HTML
  Parameters: {"locale"=>"pt"}
  KalibroConfigurationAttributes Load (0.2ms)  SELECT  "kalibro_configuration_attributes".* FROM "kalibro_configuration_attributes" WHERE "kalibro_configuration_attributes"."kalibro_configuration_id" = $1 LIMIT 1  [["kalibro_configuration_id", 4]]
  KalibroConfigurationAttributes Load (0.2ms)  SELECT  "kalibro_configuration_attributes".* FROM "kalibro_configuration_attributes" WHERE "kalibro_configuration_attributes"."kalibro_configuration_id" = $1 LIMIT 1  [["kalibro_configuration_id", 3]]
  KalibroConfigurationAttributes Load (0.2ms)  SELECT  "kalibro_configuration_attributes".* FROM "kalibro_configuration_attributes" WHERE "kalibro_configuration_attributes"."kalibro_configuration_id" = $1 LIMIT 1  [["kalibro_configuration_id", 2]]
  KalibroConfigurationAttributes Load (0.1ms)  SELECT  "kalibro_configuration_attributes".* FROM "kalibro_configuration_attributes" WHERE "kalibro_configuration_attributes"."kalibro_configuration_id" = $1 LIMIT 1  [["kalibro_configuration_id", 1]]
  Cache digest for app/views/home/index.html.erb: bd6b86e43ee3144d2e082252dd8d607d
Read fragment views/localhost:3000/pt/bd6b86e43ee3144d2e082252dd8d607d (0.1ms)
  Cache digest for app/views/home/index.html.erb: bd6b86e43ee3144d2e082252dd8d607d
Read fragment views/localhost:3000/pt/bd6b86e43ee3144d2e082252dd8d607d (0.0ms)
  Cache digest for app/views/home/index.html.erb: bd6b86e43ee3144d2e082252dd8d607d
Read fragment views/localhost:3000/pt/bd6b86e43ee3144d2e082252dd8d607d (0.1ms)
  Rendered home/index.html.erb within layouts/application (3.4ms)
  Rendered shared/_navbar.html.erb (1.8ms)
  Rendered shared/_footer.html.erb (2.1ms)
Completed 200 OK in 465ms (Views: 444.0ms | ActiveRecord: 0.7ms)
```